lib/node_modules/@stdlib/array/bool/docs/repl.txt

```diff
@@ -544,6 +577,39 @@
     true
 
 
+{{alias}}.prototype.some( predicate[, thisArg] )
+    Returns a boolean indicating whether at least one element passes a test.
+
+    A predicate function is provided the following arguments:
+
+    - value: current array element.
+    - index: current array element index.
+    - arr: the array on which the method was called.
+
+    Parameters
+    ----------
+    predicate: Function
+        Predicate function which tests array elements. If a predicate function
+        returns a truthy value, an array element passes; otherwise, an array
+        element fails.
+    
+    thisArg: Any (optional)
+        Execution context.
+    
+    Returns
+    -------
+    bool: boolean
+        Boolean indicating whether at least one element passes the test.
+    
+    Examples
+    --------
+    > function predicate( v ) { return v === true; }
```

Comment:
```suggestion
    > function predicate( v ) { return v === true; };
```

If we are not showing return values, we need to silence the commands by ending the expression with a semicolon.